addons/wardrobe/ACE_Wardrobe_BaseClasses.hpp

```diff
@@ -47,6 +47,22 @@ class GVAR(base_U_gloves_off): GVAR(base) {
     alternativeActionName = CSTRING(glovesOff);
 };
 
+// Common base class for uniforms with ties on/off variants
+class GVAR(base_U_tie_on): GVAR(base) {
```

Comment:
Used names more in line with existing (instead of 'with tie' went for 'tie on').